README.md

```diff
@@ -58,6 +58,10 @@ make help
 
 The Make files are autodocumented.
 
+### Technology
+
+It is possible to initialize an available technology design flow by running `make tech-init` after cloning the repo.
```

Comment:
If the technology is in general not free on github, what is generated by that command?